File: Selenium/Project/CRM_Activity5.java
package project;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.Color;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity5{
	

	//declare driver
		WebDriver driver;
		WebDriverWait wait;
		
		
		@BeforeClass
		  public void beforeClass() {
			  driver = new ChromeDriver();
			  wait = new WebDriverWait(driver,Duration.ofSeconds(10));
			  
		  }
		
  @Test
  public void getColor() {
	  //Login
	  CRM_Activity4 login = new CRM_Activity4();
	  login.verifyLogin(driver);
	  
	  wait.until(ExpectedConditions.visibilityOfElementLocated(By.xpath("*//div[@class='desktop-toolbar']")));

	  Color toolbarColor = Color.fromString(driver.findElement(By.cssSelector("div.desktop-toolbar")).getCssValue("color"));
	  System.out.println("The color of navigation toolbar is : "+toolbarColor.asHex());
	  
	  
	 Assert.assertEquals(toolbarColor.asHex(), "#534d64");
	  
	  
	  
	  
  }
  
  @AfterClass
  public void afterClass() {
	  
	  if (driver != null) {
          driver.quit();
  }
  }
  
}

File: Selenium/Project/CRM_Activity5.py
# Import 
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC
from CRM_Activity4 import verify_login
from selenium.webdriver.support.color import Color

# Start the Driver
with webdriver.Chrome() as driver:
    # Navigate to the URL
        driver.get("https://alchemy.hguy.co/crm/")

        verify_login(driver)
        wait = WebDriverWait(driver,5)

        wait.until(EC.visibility_of_element_located((By.CSS_SELECTOR, "div.desktop-toolbar")))

        # Get the CSS property value of the toolbar color
        toolbar_color = Color.from_string(driver.find_element(By.CSS_SELECTOR, "div.desktop-toolbar").value_of_css_property("color"))
        print("The color of the navigation toolbar is: ",toolbar_color.hex)

        assert toolbar_color.hex=="#534d64", "Color text does not match!"

        
        driver.quit()

